src/Generator.Bind/XML/Signatures/SignatureReader.cs

```diff
@@ -13,7 +13,9 @@
 namespace Bind.XML.Signatures
 {
     /// <summary>
-    /// Reads signature definitions from an XML document.
+    /// Reads API profile definitions from an XML document.
+    ///
```

Comment:
The reader reads signature definitions, not API profile definitions. Different concepts. If the class does not do that, its name should change.

Reply:
Will revert. This is a 4am special.